Bottom: dc6f9374d0e7c8f8e770832643b97a61af61592e
Top:    fb3693eefef5e9d75f323cb7d6edab9641719282
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-25 16:14:23 -0500

Refresh of add-haproxy-role

---
diff --git a/roles/haproxy/templates/haproxy.cfg.j2 b/roles/haproxy/templates/haproxy.cfg.j2
index 228ef73a0..975cc0d35 100644
--- a/roles/haproxy/templates/haproxy.cfg.j2
+++ b/roles/haproxy/templates/haproxy.cfg.j2
@@ -5,9 +5,11 @@ global
 
 defaults
     mode http
+    option forwardfor
     timeout connect 5s
     timeout client 50s
     timeout server 50s
+    timeout tunnel 3600s
     default-server init-addr last,libc,none
 
 #enable stats page
